src/viam/components/gripper/gripper.py

```diff
@@ -73,6 +78,33 @@ async def grab(
         """
         ...
 
+    @abc.abstractmethod
+    async def is_holding_something(
+        self,
+        *,
+        extra: Optional[Dict[str, Any]] = None,
+        timeout: Optional[float] = None,
+        **kwargs,
+    ) -> HoldingStatus:
+        """
+        Get information about whether the gripper is currently holding onto an object.
+
+        ::
+
+            my_gripper = Gripper.from_robot(robot=machine, name="my_gripper")
+
+            # Grab with the gripper.
+            holding_status = await my_gripper.is_holding_something()
+            # get the boolean result
+            is_holding_something = holding_status.is_holding_something
```

Comment:
(q) naively, `is_holding_something` is the only thing I care about here and I don't understand what the `meta` field does at all, so this looks to me like we're just asking for an extra step to get the relevant information. Can you clarify what the `meta` field in a `HoldingStatus` actually is?